sdcm/nemesis.py

```diff
@@ -345,6 +354,20 @@ def disrupt(self):
         self.disrupt_nodetool_decommission(add_node=True)
 
 
+class NoCorruptRepairMonkey(Nemesis):
```

Comment:
The name could've been simply `RepairMonkey`, no need to make it an opposition to the `CorruptThenRepairMonkey`. Other than that pretty small detail, this looks good.